samples/CppArExample/qml/SettingsWindow.qml

```diff
@@ -90,6 +93,18 @@ Item {
                             text: "Reset"
                             onClicked: resetTrackingClicked();
                         }
+                        Button {
+                            text: "Ignore Tracking"
```

Comment:
I don't know what standards you may have settled on for button labels. I noticed earlier today in another PR that there was a mixture of capitalizing all words in the label and capitalizing only the first word. Whatever the team decides on, you should stick with one or the other format.

Reply:
I used "random capitalization standard"... Fixed to keep the capital only the first word.